releasenotes/notes/fix-logger-glog-80cfbb7958da4c64.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Remove the flag.Parse() to use a fix avoiding to overide the logger.
```

Comment:
override

Reply:
could we make this relnote more user-oriented? (i.e. link this fix to a bug in behavior that users may have experienced earlier)

Something like `Fix setup of command-line options on the main `agent` command` (this may need re-wording too). Let me know what you think